frontend/micro-ui/web/micro-ui-internals/packages/css/src/components/radiobtn.scss

```diff
@@ -38,6 +38,15 @@
     @apply block bg-primary-main h-3 w-3 rounded-full absolute top-6 left-6;
   }
 }
+
+
+@screen sm {
+  .radio-wrap {
+    @apply flex-col;
+    gap : 0px !important;
```

Comment:
if possible, can we avoid using '!important' ?

Reply:
There are few inline css on that element. We need important in this case to override them.